static/src/stylesheets/module/_release-message.scss

```diff
@@ -939,3 +978,105 @@ $btn-height: gs-height(1) - $gs-baseline/2; //30px
     width: 100%;
     height: 100%;
 }
+
+.membership__remind-me-form {
+    border-top: 1px dotted #333333;
```

Comment:
We can use [vars for colours](http://guardian.github.io/pasteup-palette/demo/) 

`border-top: 1px dotted $neutral-1;`